client/src/api/Column.js

```diff
@@ -21,9 +21,39 @@ export const addColumnByBoardId = async (boardId, formData) => {
 // Update column
 export const updateColumn = async (columnId, updatedColumn) => {
   try {
-    const res = await axios.put(`/api/columns/${columnId}`, updatedColumn);
+    const config = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+    const resp = await axios.put(
+      `/api/columns/${columnId}`,
+      updatedColumn,
+      config,
+    );
+    // console.log(
+    //   `column ${columnId.substring(
+    //     columnId.length - 4,
+    //     columnId.length,
+    //   )} updated at ${Date(Date.now())}`,
+    // );
+    return {data: resp.data, error: false};
+  } catch (err) {
+    return {data: [], error: true};
+  }
+};
+
+// Delete Column
+export const removeColumn = async (colId) => {
+  try {
+    const res = await axios.delete(
+      `/api/columns/${colId}`,
+      {data: []},
```

Comment:
Is there a reason to pass an empty data array when calling delete? Just curious if it's needed since I thought it only needed the id.

Reply:
Oh yeah you dont need an empty data array, it was a vestige of some debugging shenanigans I had